cogs/mdoc.py

```diff
@@ -26,23 +31,36 @@ async def mdoc(self, ctx, *args):
             return
 
         try:
-            container_output = dockerclient.containers.run(
-                image="manimcommunity/manim:stable",
-                command=f"""timeout 10 python -c "import manim; assert '{arg}' in dir(manim); print(manim.{arg}.__module__ + '.{arg}')" """,
-                user=os.getuid(),
-                stderr=False,
-                stdout=True,
-                detach=False,
-                remove=True,
-            )
-        except docker.errors.ContainerError as e:
+            if config.NO_DOCKER:
+                errortype = Exception
+                proc = subprocess.run(
+                    f"""timeout 10 python -c "import manim; assert '{arg}' in dir(manim); print(manim.{arg}.__module__ + '.{arg}')" """,
+                    shell=True,
+                    stderr=subprocess.PIPE,
+                    stdout=subprocess.PIPE,
+                )
+                err = proc.stderr.decode("utf-8")
+                if err:
+                    raise Exception(err)
+            else:
+                errortype = docker.errors.ContainerError
+                container_output = dockerclient.containers.run(
+                    image="manimcommunity/manim:stable",
+                    command=f"""timeout 10 python -c "import manim; assert '{arg}' in dir(manim); print(manim.{arg}.__module__ + '.{arg}')" """,
+                    user=os.getuid(),
+                    stderr=False,
+                    stdout=True,
+                    detach=False,
+                    remove=True,
+                )
+        except errortype as e:
             if "AssertionError" in e.args[0]:
                 await ctx.reply(f"I could not find `{arg}` in our documentation, sorry.")
                 return
             await ctx.reply(f"Something went wrong: ```{e.args[0]}```")
             return
 
-        fqname = container_output.decode("utf-8").strip().splitlines()[2]
+        fqname = (proc.stdout.decode("utf-8").strip().splitlines()[0] if config.NO_DOCKER else container_output.decode("utf-8").strip().splitlines()[2])
```

Comment:
```suggestion
        if config.NO_DOCKER:
            fqname = proc.stdout.decode("utf-8").strip().splitlines()[0]
        else:
            fqname = container_output.decode("utf-8").strip().splitlines()[2]
```

Again, being a bit more explicit does not hurt here.